tests/test_find_new_matches.py

```diff
@@ -59,22 +65,28 @@ def test_tf_tables_init_works():
         )
 
 
-def test_matches_work():
-    linker = DuckDBLinker(
-        df,
-        settings,
-    )
+@mark_with_dialects_excluding()
+def test_matches_work(test_helpers, dialect):
+    helper = test_helpers[dialect]
+    Linker = helper.Linker
+    brl = helper.brl
+    df = helper.load_frame_from_csv("./tests/datasets/fake_1000_from_splink_demos.csv")
```

Comment:
convert to in-built dataset

Reply:
Do we definitely want to? The in-built dataset is downloaded, so will add to the total run time of the tests.

Reply:
Unless you mean something else

Reply:
Good point - I hadn't thought about the additional runtime. Feel free to leave for now and we an have a more general conversation about how we use data for testing when you are back from leave. The key thing is that everything passes, which they do